Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingDll.cs

```diff
@@ -492,59 +447,33 @@ public static void GeneratePlayspace_RequestFinish()
             /// <param name="bufferIndexCount">Size of indices, in number of elements</param>
             /// <param name="indices">Array to receive the mesh indices</param>
             /// <returns>Zero if fails, one if success</returns>
-#if UNITY_METRO && !UNITY_EDITOR
             [DllImport("SpatialUnderstanding")]
             public static extern int GeneratePlayspace_ExtractMesh_Extract(
                 [In] int bufferVertexCount,
                 [In] IntPtr verticesPos,        // (vertexCount) DirectX::XMFLOAT3*
                 [In] IntPtr verticesNormal,     // (vertexCount) DirectX::XMFLOAT3*
                 [In] int bufferIndexCount,
                 [In] IntPtr indices);           // (indexCount) INT32*
-#else
-            public static int GeneratePlayspace_ExtractMesh_Extract(
-                [In] int bufferVertexCount,
-                [In] IntPtr verticesPos,        // (vertexCount) DirectX::XMFLOAT3*
-                [In] IntPtr verticesNormal,     // (vertexCount) DirectX::XMFLOAT3*
-                [In] int bufferIndexCount,
-                [In] IntPtr indices)            // (indexCount) INT32*
-            {
-                return 0;
-            }
-#endif
 
             /// <summary>
             /// Query the playspace scan statistics. 
             /// </summary>
             /// <param name="playspaceStats">playspace stats structure to receive the statistics data</param>
             /// <returns>Zero if fails, one if success</returns>
-#if UNITY_METRO && !UNITY_EDITOR
             [DllImport("SpatialUnderstanding")]
             public static extern int QueryPlayspaceStats(
+                // TODO: investigate... this sure looks like an "out"
```

Comment:
Thanks, @jevertt .  Maybe the In/Outs only matter for more complex marshalling situations (like COM marshalling across processes/machines).  I'm just going to remove the TODOs for now.